server/routes/nl_interface.py

```diff
@@ -343,7 +343,9 @@ def page(dcid):
   chart_config = _chart_config(place_dcid, main_place_type, main_place_name,
                                child_places_type, highlight_svs, sv2name,
                                peer_buckets)
-
-  print(chart_config)
-
-  return render_template('/nl.html')
+  message = ParseDict(chart_config, subject_page_pb2.SubjectPageConfig())
```

Comment:
maybe I'm too greedy :)  is it a trivial change to pass in a query param in place of hardcoding in line 322?  then we can test out different queries

Reply:
Yea, added a query param